sphinxserve/__init__.py

```diff
@@ -6,21 +6,25 @@
 
 __version__ = '0.7.4'
 __author__ = 'Daniel Mizyrycki'
```

Comment:
Lets not change style for the sake of style. In particular changing the import order of gevent can and will introduce bugs.